ARExperiment/ARExperiment/Model/ARViewModel.swift

```diff
@@ -33,11 +41,8 @@ class ARViewModel {
         return hitResultsFeaturePoints.first
     }
 
-    private func createAssetPath(assetFolder: String?, fileName: String, fileExtension: String) -> String {
-        if let assetFolder = assetFolder {
-            return "art.scnassets/\(assetFolder)/\(fileName).\(fileExtension)"
-        }
-        return "art.scnassets/\(fileName).\(fileExtension)"
+    func nodeExistOnScene(_ sceneView: ARSCNView, nodeName: String) -> SCNNode? {
```

Comment:
perhaps this would be clearer named:

```suggestion
    func node(in sceneView: ARSCNView, named nodeName: String) -> SCNNode? {
```